### DISCUSSION TITLE
Automatic port selection

### DISCUSSION THREAD

**Discussion by An88ton** [uNetworking/uWebSockets.js#557] [Q&A] [answered]
Hi, I have a question about this example https://github.com/uNetworking/uWebSockets.js/blob/9b3efbdfa315f4ec559fc8a6afe78083007bb7df/examples/AutomaticPortSelection.js
On version 19.2 it does not work, I get this error "TypeError: uws.us_socket_local_port is not a function". So will it be fixed, and if yes then when?

**Comment by ghost** [comment, marked as answer]
Yes this is a new function that will be released in next version

**Reply by An88ton** [reply]
Thank you for the reply
